src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/ActiveDirectoryAuthenticationProvider.cs

```diff
@@ -545,14 +583,15 @@ private IPublicClientApplication CreateClientAppInstance(PublicClientAppKey publ
             IPublicClientApplication publicClientApplication;
 
 #if NETFRAMEWORK
```

Comment:
With the comments above, a lot of these conditional compilations can probably be removed.